lmms_eval/models/qwen_vl.py

```diff
@@ -228,21 +228,25 @@ def _collate(x):
                     until = [until]
                 elif not isinstance(until, list):
                     raise ValueError(f"Expected `gen_kwargs['until']` to be of type Union[str,list] but got {type(until)}")
+
+            if isinstance(contexts, tuple):
+                contexts = list(contexts)
+
             for i in range(len(contexts)):
                 if "<image>" in contexts[i]:
-                    context[i] = contexts[i].replace("<image>", "")
-            questions = [self.prompt.format(visual_path, context) for visual_path, context in zip(visual_paths, contexts)]
```

Comment:
removed this line since `questions` is being assigned below in the `questions = self.tokenizer.from_list_format(query)` line.